DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -86,20 +87,56 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
 
     @MainActor
     private func enableNetP() async {
-        do {
-            try await tunnelController.setState(to: true)
-        } catch {
-            statusMessage = error.localizedDescription
-            isNetPEnabled = false
-        }
+        await tunnelController.start()
     }
 
     @MainActor
     private func disableNetP() async {
-        do {
-            try await tunnelController.setState(to: false)
-        } catch {
-            statusMessage = error.localizedDescription
+        await tunnelController.stop()
+    }
+
+    private class func titleText(connected isConnected: Bool) -> String {
+        isConnected ? UserText.netPStatusHeaderTitleOn : UserText.netPStatusHeaderTitleOff
+    }
+
+    private class func statusImageID(connected isConnected: Bool) -> String {
+        isConnected ? "VPN" : "VPNDisabled"
+    }
+}
+
+private extension ConnectionStatus {
+    var message: String {
+        switch self {
+        case .notConfigured:
+            return "Not Configured"
```

Comment:
Just a note that at least in macOS we don't show this status even though it exists... we show `.disconnected`.

I don't think it needs to block this PR but something worth synching with between you, Robert and myself maybe?

Reply:
Also these strings should probably be `LocalizedString` (since they're using facing)?

I know SwiftUI handles that on its own, so I'm not sure whether this is needed or optional for iOS.  Please let me know so I can learn about it!